Library/Homebrew/utils/analytics.rb

```diff
@@ -16,6 +16,8 @@ module Analytics
     INFLUX_HOST = "https://eu-central-1-1.aws.cloud2.influxdata.com"
     INFLUX_ORG = "d81a3e6d582d485f"
 
+    extend Cachable
+
     class << self
       include Context
```

Comment:
We could potentially remove the `class << self` here for consistency but that would blow up the size of the diff a lot.

Reply:
@apainintheneck Worth doing in follow-up PR! In general I think it's not worth blowing up size of diff if disabling whitespace changes can hide it 👍🏻